packages/react-components/src/components/knowledge-graph/KnowledgeGraphPanel.tsx

```diff
@@ -39,6 +40,38 @@ export const KnowledgeGraphContainer: React.FC<KnowledgeGraphInterface> = ({
   queryData,
   ...props
 }) => {
+  const defaultLayoutOptions = {
+    spacingFactor: 0.4,
+    animate: false,
+    fit: false,
+    padding: 30,
+    nodeDimensionsIncludeLabels: false,
+    randomize: false,
+    componentSpacing: 40,
+    nodeRepulsion: function (_node: any) {
+      return 2048;
+    },
+    nodeOverlap: 4,
+    edgeElasticity: function (_edge: any) {
+      return 32;
+    },
+    nestingFactor: 1.2,
+    gravity: 1,
+    initialTemp: 1000,
+    coolingFactor: 0.99,
+    minTemp: 1.0,
+  };
+  const breadthFirstlayout = {
+    ...defaultLayoutOptions,
+    name: 'breadthfirst',
+  };
+  const coselayout = {
+    ...defaultLayoutOptions,
+    name: 'grid',
+    fit: true,
+    padding: 50,
+  };
+
```

Comment:
working on moving this to a separate layout file for better and clean code, as of now if i do so and import the layout it doesn't work and layout doesn't get applied unless its in the same file(very strange).I am working on a fix but I need this to ship before private beta.